.github/workflows/test-and-publish.yml

```diff
@@ -41,3 +57,14 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           build-args: |
             GITHUB_REF_NAME=${{ github.ref_name }}
+
+      - name: Build and push Docker image to GitHub Container Registry
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          tags: |
+            ghcr.io/${{ github.repository }}:${{ github.ref_name }}
+          labels: ${{ steps.meta.outputs.labels }}
+          build-args: |
+           GITHUB_REF_NAME=${{ github.ref_name }} 
```

Comment:
```suggestion
            GITHUB_REF_NAME=${{ github.ref_name }} 
```

Reply:
yes, i made a mistake in that area. so i corrected using space instead of using tab. Mybad.